.gitignore

```diff
@@ -10,3 +10,8 @@ packages/docs/.vitepress/.cache/
 packages/docs/.vitepress/.temp/
 packages/docs/.vitepress/dist/
 packages/oruga/README.md
+packages/docs-next/.vitepress/cache/
```

Comment:
the package name got changed from docs-next to docs, as well as oruga-next to next. So no need for this adding.

Reply:
Clearly I need more coffee. You're right, i just had leftover files in the old directories locally.